test/SourceKit/Indexing/Inputs/test_module.index.response

```diff
@@ -175,6 +156,11 @@
       key.kind: source.lang.swift.decl.function.free,
       key.name: "globalFunc()",
       key.usr: "s:11test_module10globalFuncyyF"
+    },
+    {
+      key.kind: source.lang.swift.decl.extension.class,
+      key.name: "C2",
+      key.usr: "s:e:s:11test_module2C2C6SECRETyyF"
```

Comment:
This change does mean extensions might exist with USRs that refer to decls that aren't indexed. Is that a problem?